modules/cnf-programming-the-sriov-operator.adoc

```diff
@@ -0,0 +1,398 @@
++
+[NOTE]
+====
+The role of Forward Error Correction (FEC) is to correct transmission errors, where certain bits in a message can be lost or garbled. Messages can be lost or garbled due to noise in the transmission media, interference, or low signal strength.
+Without FEC, a garbled message would have to be resent, adding to the network load and impacting throughput and latency.
+====
+
+.Prerequisites
+
+* Intel® FPGA PAC N3000 card
+* Node or nodes installed with the OpenNESS Operator for Intel® FPGA PAC N3000 (Programming)
+* Node or nodes installed with the OpenNESS Operator for Wireless FEC Accelerators
+* RT kernel configured with Performance Addon Operator
++
+[NOTE]
+====
+All the commands run in the `vran-acceleration-operators` namespace.
+====
+
+.Procedure
+
+. Verify that the SR-IOV-FEC Operator is installed and that pods are running:
++
+[source,terminal]
+----
+$ oc get csv -o custom-columns=Name:.metadata.name,Phase:.status.phase
+----
++
+.Example output
+[source,terminal]
+----
+Name                                        Phase
+sriov-fec.v1.1.0                            Succeeded
+n3000.v1.1.0                                Succeeded
+----
+
+. Verify that the pods are running:
++
+[source,terminal]
+----
+$  oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY       STATUS      RESTARTS    AGE
+fpga-driver-daemonset-8xz4c                     1/1         Running     0           15d
+fpgainfo-exporter-vhvdq                         1/1         Running     1           15d
+N3000-controller-manager-b68475c76-gcc6v        2/2         Running     1           15d
+N3000-daemonset-5k55l                           1/1         Running     1           15d
+N3000-discovery-blmjl                           1/1         Running     1           15d
+N3000-discovery-lblh7                           1/1         Running     1           15d
+sriov-device-plugin-j5jlv                       1/1         Running     1           15d
+sriov-fec-controller-manager-85b6b8f4d4-gd2qg   1/1         Running     1           15d
+sriov-fec-daemonset-kqqs6                       1/1         Running     1           15d
+----
+The following section provides information on the installed pods:
+* `fpga-driver-daemonset` provides and loads the required Open Programmable Accelerator Engine (OPAE) drivers
+* `fpgainfo-exporter` provides N3000 telemetry data for Prometheus
+* `N3000-controller-manager` applies N3000Node CRs to the cluster and manages all the operand containers
+* `N3000-daemonset` is the main worker application
+* `N3000-discovery` discovers N3000 Accelerator devices installed and labels worker nodes if devices are present
+* `sriov-device-plugin` expose the FEC virtual functions as resources under the node
+* `sriov-fec-controller-manager` applies CR to the node and maintains the operands containers
+* `sriov-fec-daemonset` is the main worker application
```

Comment:
Ideally, you can repeat the more precise description from other files and replace "main worker application" in these two instances.